File: app/src/main/java/eu/toma/dev/playground/warcraft/model/Talent.java
package eu.toma.dev.playground.warcraft.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Talent {

    @SerializedName("tier")
    @Expose
    public Integer tier;
    @SerializedName("column")
    @Expose
    public Integer column;
    @SerializedName("spell")
    @Expose
    public Spell spell;
    @SerializedName("spec")
    @Expose
    public Spec spec;
}

File: app/src/main/kotlin/warcraft/model/Talent.kt
package warcraft.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Talent {

    @SerializedName("tier")
    @Expose
    var tier: Int? = null
    @SerializedName("column")
    @Expose
    var column: Int? = null
    @SerializedName("spell")
    @Expose
    var spell: Spell
    @SerializedName("spec")
    @Expose
    var spec: Spec
}